test/integration/gce-mig/gce_mig_regional_lb_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gce_mig_regional_lb
+
+import (
+	"testing"
+
+	"io"
+	"net/http"
+	"time"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft"
+	"github.com/stretchr/testify/assert"
+)
+
+func AssertResponseStatus(t *testing.T, assert *assert.Assertions, url string, statusCode int) {
```

Comment:
Could we reusing existing helpers?

https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/b3556b0aa83041b4e4b6f9d2131cdfceaa3a3b72/infra/blueprint-test/pkg/utils/asserthttp.go#L91

Reply:
* Used existing `AssertResponse` from the helper. https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/b3556b0aa83041b4e4b6f9d2131cdfceaa3a3b72/infra/blueprint-test/pkg/utils/asserthttp.go#L105
* `AssertResponseWithRetry` was not honoring retry interval. Would required some deep dive to figure out the reason.